packages/cli/README.md

```diff
@@ -1,32 +1,31 @@
 # Ultralight CLI
 
-This is an experimental tool for running an ultralight node from the command line in a NodeJS environment.  All functionality for the portal module behaves in the same manner as the [browser client](../browser-client).
+This is an experimental implementation an Ultralight node packaged as a NodeJS application and run from the command line.  
 
 ## Usage
 
-### Quickstart
-- Clone the ultralight monorepo and run `npm i` from the monorepo root.
-- Change to `./packages/cli`
-- Run `node dist/index.js --proxy --nat=extip --bootnode=enr:[Your favorite bootnode ENR here (if desired)] `
-- The node will start a local instance of the network proxy, determine its public IP address, and then connect to the bootnode specified
+The easiest way to get started with Ultralight is to use the `npm run dev` script to start up a a single local node:
+
+`npm run dev` along with any of the below parameters.
```

Comment:
I would personally prefer some command name a bit more expressive here.

Is `ultralight` too much of an "approximation"? I would personally like and this would feasible as a "semantic simplification"? No very strong opinion.

So: `npm run ultralight`

Reply:
Update: ok, but I understand also if/that you want to make sure that it is clear that these are not the final networks.